usr/share/rear/verify/VEEAM/default/400_verify_veeam.sh

```diff
@@ -13,6 +13,9 @@ fi
 Log "Deleting Veeam SQLite Linux Agent Database"
 rm -rf /var/lib/veeam/*
 
```

Comment:
Please add a comment that tells why that is done here, cf.
https://github.com/rear/rear/wiki/Coding-Style#code-must-be-easy-to-understand-answer-the-why
e.g. somwething like
```
# mount the /tmp directory with tmpfs option,
# otherwise veeammount command will fail with versions 12.1 and above
# see https://github.com/rear/rear/pull/3213